docs/dev-guide/dev_plugin_code.md

```diff
@@ -1,340 +1,343 @@
-# 插件代码开发手册
+# Plugin Code Development Guide
 
-本文档主要针对**Sermant**的[示例模块](../../sermant-plugins/sermant-example)，介绍插件开发过程中常见的一些场景。
+[简体中文](dev_plugin_code-zh.md) | [English](dev_plugin_code.md) 
 
-- [组成部分](#组成部分)
-- [插件模块](#插件模块)
-  - [增强定义](#增强定义)
-    - [原生类增强](#原生类增强)
-  - [拦截器](#拦截器)
-  - [插件配置](#插件配置)
-  - [插件服务](#插件服务)
-    - [简单插件服务](#简单插件服务)
-    - [复杂插件服务](#复杂插件服务)
-    - [日志功能](#日志功能)
-    - [心跳功能](#心跳功能)
-    - [链路功能](#链路功能)
-    - [动态配置功能](#动态配置功能)
-- [插件服务模块](#插件服务模块)
+This document focuses on **Sermant**'s [example module](../../sermant-example), which describes some common scenarios during plug-in development.
 
-## 组成部分
+- [Components](#Components)
+- [Plugin Module](#Plugin-Module)
+  - [Enhancement Definition](#Enhancement-Definition)
+    - [Enhancement for Native Class](#Enhancement-for-Native-Class)
+  - [Interceptor](#Interceptor)
+  - [Plugin Configuration](#Plugin-Configuration)
+  - [Plugin Service](#Plugin-Service)
+    - [Simple Plugin Service](#Simple-Plugin-Service)
+    - [Complex Plugin Service](#Complex-Plugin-Service)
+    - [Log](#Log)
+    - [Heartbeat](#Heartbeat)
+    - [Trace Tracking](#Trace-Tracking)
+    - [Dynamic Configuration](#Dynamic-Configuration)
+- [Plugin Service Module](#Plugin-Service-Module)
 
-由[插件模块开发手册](dev_plugin_module.md)可知，一个`插件主模块(main)`中可能包含以下5种内容：
+## Components
 
-- `插件模块(plugin)`，该模块主要用于声明对宿主应用的增强逻辑
-- `服务模块(service)`，用于为插件包提供服务实现
-- `后端模块(server)`，用于接收插件数据的服务端
-- `前端模块(webapp)`，用于对服务端数据作前端展示
-- `其他模块(other)`，特殊附加件，一般用作调试
+According to the [Plugin Module Development Guide](dev_plugin_module.md), a `main` plugin module contains the following five things:
```

Comment:
Plugin Module Development Guide   is different to it's zh version which release to add

Reply:
resolved